### ISSUE TITLE
Problems with too big users data.json-File

### ISSUE THREAD

**Issue by AnthyG** [AnthyG/ThunderWave#26]
In the following image, there are 3 different accounts used, the big one on the left has quite a huge data.json-File already.
This seems to cause some problems in 'syncing' the file to the two other peers, as they can see each others messages!

I believe, this can be fixed through creating a Hub (or even several!) for ThunderWave, which would then contain all the data.

![image](https://cloud.githubusercontent.com/assets/15957203/24932903/f1e644f4-1f13-11e7-8846-af432c6c97ae.png)

**Comment by Plasmmer** [comment]
You can make messages as optional files, and users can seed messages.

**Comment by AnthyG** [comment]
I'm not sure, if that's such a good idea.
But the other way, maybe this could be good for blocking users.
The user would simply not have the blocked-users messages on his disk.
And if a user is blocked by everybody, he'll most-likely be a spammer or similar 😈

This would probably require a reformatting of the Database/ `data.json`-File, meaning, I would probably have to override _**all**_ the messages that have been written in the lobby... Which are _quite_ a lot!

**Comment by Plasmmer** [comment]
Not. Attackers will discover it and create a lot of accounts to ban a user. I'll create a new issue about it.

**Comment by Plasmmer** [comment]
Even blocking spammers, ThunderWave will have a lot of messages, so seeding is needed.

**Comment by Plasmmer** [comment]
#71 will be needed in order to ban a user based on votes of VERIFIED users. But I say again, it will not solve disk space issues.

**Comment by DaniellMesquita** [comment]
@AnthyG
